cuda_python/docs/source/index.rst

```diff
@@ -6,8 +6,8 @@ multiple components:
 
 - `cuda.core`_: Pythonic access to CUDA runtime and other core functionalities
 - `cuda.bindings`_: Low-level Python bindings to CUDA C APIs
-- `cuda.cooperative`_: Pythonic exposure of CUB cooperative algorithms
-- `cuda.parallel`_: Pythonic exposure of Thrust parallel algorithms
+- `cuda.cooperative`_: Exposure of CUB cooperative algorithms for use in Python kernels
+- `cuda.parallel`_: Pythonic exposure of Thrust parallel algorithms launched from the host
```

Comment:
Should we adopt the new README.md wording here, too?

Reply:
Adopted.